_books/ion-1-1/src/macros/system_macros.md

```diff
@@ -297,19 +315,22 @@ Examples:
 (:sum (:)) => 0
 ```
 
-#### `comment`
+#### `meta`
 
 ```ion
-(macro comment (anything*) (.none))
+(macro meta (anything*) (.none))
 ```
 
-The `comment` macro accepts any values and emits nothing.
-It can be used for comments that must be syntactically valid Ion.
+The `meta` macro accepts any values and emits nothing.
+It allows writers to encode data that will be not be surfaced to most readers.
+Readers can be configured to intercept calls to `meta`, allowing them to read the otherwise invisible data.
+
+When transcribing from one format to another, writers should preserve invocations of `meta` when possible.
 
 Example:
 ```ion
 (:values
-    (:comment equivalent to {foo:1,foo:2})
+    (:meta {author: "Mike Smith", email: "mikesmith@example.com"})
```

Comment:
🗺️ `comment` is an unnecessarily narrow name. This macro can accept arbitrary data.